src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -155,6 +160,80 @@ private void OnGUI()
             // project = EditorGUILayout.TextField("Project", project);
             // EditorGUILayout.EndToggleGroup();
         }
+
+        /// <summary>
+        /// Creates Sentry folder for storing its configs - Assets/Resources/Sentry
+        /// </summary>
+        private static void TryCreateSentryFolder()
+        {
+            // TODO: revise, 'Resources' is a special Unity folder which is created by default. Not sure this check is needed.
+            if (!AssetDatabase.IsValidFolder("Assets/Resources"))
+            {
+                AssetDatabase.CreateFolder("Assets", "Resources");
+            }
+            if (!AssetDatabase.IsValidFolder($"Assets/Resources/{UnitySentryOptions.ConfigRootFolder}"))
+            {
+                AssetDatabase.CreateFolder("Assets/Resources", UnitySentryOptions.ConfigRootFolder);
+            }
+        }
+
+        /// <summary>
+        /// Find and copy 'link.xml' into current Unity project for IL2CPP builds
+        /// </summary>
+        private static void TryCopyLinkXml()
+        {
+            const string linkXmlFileName = "link.xml";
+
+            var linkXmlPath = $"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/{linkXmlFileName}";
+            if (File.Exists(linkXmlPath))
+            {
+                // TODO: should we log here?
+                return;
+            }
+
+            var linkPath = GetLinkXmlPath(linkXmlFileName);
+            if (linkPath == null)
+            {
+                // TODO: should we log here?
+                return;
+            }
+
+            var linkXmlAsset = AssetDatabase.LoadAssetAtPath<TextAsset>(linkPath);
+            File.WriteAllBytes(linkXmlPath, linkXmlAsset.bytes);
+        }
+
+        /// <summary>
+        /// Get Unity path to 'link.xml' file from `Packages` folder
+        /// </summary>
+        private static string? GetLinkXmlPath(string linkXmlFileName)
+        {
+            var assetIds = AssetDatabase.FindAssets("io.sentry.unity", new [] { "Packages" });
+            for (var i = 0; i < assetIds.Length; i++)
+            {
+                var assetName = AssetDatabase.GUIDToAssetPath(assetIds[i]);
+                if (assetName.Contains("Runtime"))
+                {
+                    var linkFolderPath = ExtractLinkFolderPathFromEnd(assetName);
+                    return $"{linkFolderPath}/{linkXmlFileName}";
+                }
+            }
+
+            return null;
+
+            static string ExtractLinkFolderPathFromEnd(string value)

```

Comment:
`Path.GetFileName` ?

Reply:
Ah.. Yeah, I need `Path.GetDirectoryName()` in this case. Will change)